src/main/java/com/lithium/flow/access/prompts/AWSPrompt.java

```diff
@@ -0,0 +1,56 @@
+package com.lithium.flow.access.prompts;
+
+import com.lithium.aws.parameterstore.ParameterStore;
+import com.lithium.flow.access.Prompt;
+
+import javax.annotation.Nonnull;
+import java.util.Map;
+
+public class AWSPrompt implements Prompt {
+    private final String awsRegion;
+    private final String ssmParameterPath;
+    private final String configProfile;
+    private final String credentialProfile;
+
+    public AWSPrompt(@Nonnull String ssmParameterPath, @Nonnull String awsRegion, String credentialProfile, String configProfile) {
+        this.awsRegion = awsRegion;
+        this.ssmParameterPath = ssmParameterPath;
+        this.configProfile = configProfile;
+        this.credentialProfile = credentialProfile;
+    }
+
+    @Nonnull
+    @Override
+    public Response prompt(@Nonnull String name, @Nonnull String message, @Nonnull Type type) {
+        return new Response() {
+            @Nonnull
+            @Override
+            public String value() {
+                ParameterStore parameterStore = new ParameterStore(ssmParameterPath, awsRegion, credentialProfile, configProfile);
+                Map<String, Object> paramMap = parameterStore.getParamByPath();
+
+                if (name.startsWith("key[")) {
+                    String sshKey = (String) paramMap.get("sshKey");
```

Comment:
a little tighter syntax, just return `(String) paramMap.get("sshKey");`, same for `sshPassword`